pyproject.toml

```diff
@@ -35,6 +35,7 @@ dependencies = [
 [project.entry-points."sphinx.html_themes"]
 qiskit = "qiskit_sphinx_theme"
 qiskit_sphinx_theme = "qiskit_sphinx_theme"
+_qiskit-ecosystem = "qiskit_sphinx_theme"
```

Comment:
Maybe the theme name should be all underscores like `qiskit_sphinx_theme`? And why does it have a leading underscore? It's not meant to be an internal-only theme. Or are you planning to update the name once it's ready?

Reply:
> Or are you planning to update the name once it's ready?

Exactly

> Maybe the theme name should be all underscores like qiskit_sphinx_theme? 

Yeah, I wasn't sure about this either. It's part of why I wanted to keep things private for now hah. It's incredibly easy to change and not important to decide until we make things public.

I'll bring it up in the docs squad once we're ready to make things public.